### DISCUSSION TITLE
Err: index -13 is out of bounds for axis 2 with size 12! (my_folder.zip)

### DISCUSSION THREAD

**Discussion by Jiangyi-Zhu** [PaddlePaddle/PaddleSpatial#227] [BaiduKDDCup2022]
Hi committee:

This is zealen and we are having this problem as stated above, could you please kindly provide the backend detail of this error? For example, which line and which file did the error occur? Thank you very much for the help!

**Comment by ramber1836** [comment]
Please provide your team name and submission time.
